### PR TITLE
doc/tutorial: add link to the LXD UI tutorial

### PR DESCRIPTION
**Pull request by ru-fu**

### CODE REVIEW

**Review by roosterfish — APPROVED** [review]
LGTM!

**Comment by roosterfish** [comment]
Worth doing a rebase with the latest doc additions just in case?

**Comment by ru-fu** [comment]
> I guess not related to this PR but the link checker is failing. Does it need to be https://canonical-microcloud.readthedocs-hosted.com/en/latest/microcloud/doc-cheat-sheet-myst/ instead?

Ah yes, of course - the link checker checks the files that are currently online. So the integration PR moved the file, but the old file was still online, so the check didn't fail.
Will add a commit to fix this. :)

File: doc/how-to/remove_machine.md
Comment (ru-fu) [inline doc/how-to/remove_machine.md:10]: This one goes to the page now instead of the anchor. Let me add an anchor to the LXD docs. (Shouldn't block merging though, we can update this later ;) ).

**Comment by ru-fu** [comment]
I was too quick here - the LXD build hasn't completed yet, that's why the check is failing. ;)

**Comment by roosterfish** [comment]
> I was too quick here - the LXD build hasn't completed yet

Isn't the doc build pulling the LXD `main` branch so as soon as it is merged the latest changes on LXD will be available here too?

**Comment by ru-fu** [comment]
> > I was too quick here - the LXD build hasn't completed yet
> 
> Isn't the doc build pulling the LXD `main` branch so as soon as it is merged the latest changes on LXD will be available here too?

Yes - but the spell check runs a single MicroCloud build (because we only want to check the MicroCloud docs and not the other doc sets). And the single build uses the inventory file from the LXD docs that are online - and this one wasn't updated yet, so the build failed.

**Review by roosterfish — APPROVED** [review]

**Pull request merged**
